.travis.yml

```diff
@@ -1,6 +1,8 @@
 language: java
 dist: trusty
-jdk: oraclejdk8
+jdk: 
```

Comment:
Specifying both JDKs means that we will run all tests for all languages in both runtimes; which also means doubling the number of tests.  Is there a way similar to how Asset Share Commons is defined such that we just run one test in JDK11 or, since that's the direction, run all but one test in JDK11 and leave one test on JDK8 for backwards compatibility assurance?

Reply:
Good idea. Let me figure it how to achieve it. This was just the fastest way to get test coverage with Java 11.